osf_tests/factories.py

```diff
@@ -28,6 +30,7 @@
 from addons.osfstorage.models import OsfStorageFile
 
 fake = Factory.create()
+fake_email = lambda: '{}+{}@{}'.format(FAKE_EMAIL_NAME, int(time.time() * 1000000), FAKE_EMAIL_DOMAIN)
```

Comment:
Interestingly, on some systems, time.time() may resolve to the 1s precision. I presume that nothing that we'll do will do that, but it might be worth making a note here in case someone, uh, decides to run the test suite on a… windows 95 computer? A 16 bit processor? A Commodore Vic-20?

Reply:
Based on https://stackoverflow.com/a/18170448 it might be worthwhile to use `time.clock()` instead. We don't really need anything that's unique between runs, we just need it unique during the run, and `time.clock()` sounds like it might be more appropriate in this case.